src/routes/posts.js

```diff
@@ -78,9 +78,37 @@ const deletePost = async (req, res, next) => {
     }
 }
 
+// UPDATE AN ARTICLE
+
+const updatePost = async (req, res, next) => {
+    const { id } = req.params // WHERE
+    const { title, content, image, published } = req.body // SET
+    try {
+        const updateArticle = await db.query(
+            'UPDATE posts SET title = $1, content = $2 , image = $3 , published = $4 WHERE id = $5 RETURNING *',
+            [title, content, image, published, id]
+        )
+        const updatedArticle = updateArticle.rows[0]
```

Comment:
@beblicarl what if the article didn't exist, as in the id was wrong, what happens in such a scenario with our current code?

Reply:
i was assuming next(err) will throw an error to the frontend, wouldn't it?

Reply:
haha assuming, why not try a non-existent id so you share what the response is?

Reply:
alright

Reply:
this is what i got

![image](https://user-images.githubusercontent.com/68056104/185673419-dc5ad44d-3ae7-4c95-b811-7ee56e0d1b16.png)

Reply:
cool, yea also wanted the response you get from postman

Reply:
> cool, yea also wanted the response you get from postman

Error. Cannot read property of undefined.
I made new modification and now i get a proper message from postman, please take a look

![image](https://user-images.githubusercontent.com/68056104/185675710-a0055ce2-81a6-407f-82dc-59b215bc8c6f.png)

Reply:
okay, taking a look

Reply:
thanks